src/entities/monitor-system.entity.ts

```diff
@@ -17,7 +24,7 @@ export class MonitorSystem extends BaseEntity {
 
   @Column({ type: 'varchar', length: 45, nullable: false, name: 'mon_loc_id' })
   locationId: string;
-  nsps4tCompliancePeriods
+  nsps4tCompliancePeriods;
```

Comment:
Same here nsps4tCompliancePeriods isn't a column on monitor system, this can probably be removed.